apps/sensenet/src/components/view-controls/permission-view.tsx

```diff
@@ -131,6 +132,25 @@ export const PermissionView: React.FC<PermissionViewProps> = (props) => {
     visitorEntries && visitorEntries.length > 0 ? setIsPrivate(false) : setIsPrivate(true)
   }, [permissions])
 
+  const selectMemberFunction = (newEntry: EntryType) => {
+    if (permissions) {
+      const localPermissions = { ...permissions }
+      localPermissions.entries.push(newEntry)
+    }
+    setSelectedListItem(`${newEntry.identity.id}${newEntry.propagates}`)
+    openDialog({
+      name: 'permission-editor',
+      props: {
+        entry: newEntry,
+        path: currentContent!.Path,
+        callBackFunction: () => {
+          setRefreshFlag(!refreshFlag)
```

Comment:
use two parameters insted: submitCallback, cancelCallback
submitCallback is the current callbackFunction
cancelCallback: () => permission.entries.pop()